README.md

```diff
@@ -87,9 +99,34 @@ pack install-app katla
 If you no longer require *katla* and want to remove it, run
 
 ```sh
-pack remove katla
+pack remove-app katla
 ```
 
+### Updating Idris2, pack, and packages
+
+To update pack's Idris2 installation:
+
+```sh
+pack switch latest
+```
+
+Note that because packages are tied to Idris2 versions, if you update
+Idris2 then you may need to update your packages too.
```

Comment:
This section should be reworded. `pack switch latest` moves to the latest pack collection, which will

- Update the Idris2 compiler and core libraries if the latest package collection contains a newer version of them
- Update all other installed packages